Makefile

```diff
@@ -147,3 +147,33 @@ create-metrics-svc:
 .PHONY: delete-metrics-svc
 delete-metrics-svc:
 	kubectl delete -f deploy/example/metrics/csi-blob-controller-svc.yaml --ignore-not-found
+
+# compile .proto file to go output
+.PHONY: gen-proto
```

Comment:
this is for dev purpose, is it possible to put it in Readme under blobfuse-proxy dir?

Reply:
same for: 
 - clean-proto